Documentation/configuration.md

```diff
@@ -149,7 +149,110 @@ In `/etc/rkt/auth.d/specific-tectonic.json`:
 }
 ```
 
-The result is that when downloading data from `kubernetes.io` we still
-send `Authorization: Bearer common-token`, but when downloading from
-`coreos.com` - `Authorization: Basic Zm9vOmJhcg==`. And for
-`tectonic.com` - `Authorization: Bearer tectonic-token`.
+The result is that when downloading data from `kubernetes.io` `rkt`
+still sends `Authorization: Bearer common-token`, but when downloading
+from `coreos.com` - `Authorization: Basic Zm9vOmJhcg==` (`foo:bar`
+encoded in base64). And for `tectonic.com` - `Authorization: Bearer
+tectonic-token`.
+
+### rktKind: `dockerAuth`
+
+This kind of configuration is used to set up necessary credentials
+when downloading data from docker indices. The configuration files
+should be placed inside either `auth.d` or `docker.d` subdirectories
```

Comment:
Anything in mind?
Am 17.04.2015 12:21 schrieb "Krzesimir Nowak" <notifications@github.com>:

> In Documentation/configuration.md
> <https://github.com/coreos/rkt/pull/767#discussion_r28622600>:
>
> > @@ -149,7 +149,110 @@ In `/etc/rkt/auth.d/specific-tectonic.json`:
> >  }
> >  ```
> >
> > -The result is that when downloading data from `kubernetes.io` we still
> > -send `Authorization: Bearer common-token`, but when downloading from
> > -`coreos.com` - `Authorization: Basic Zm9vOmJhcg==`. And for
> > -`tectonic.com` - `Authorization: Bearer tectonic-token`.
> > +The result is that when downloading data from `kubernetes.io` `rkt`
> > +still sends `Authorization: Bearer common-token`, but when downloading
> > +from `coreos.com` - `Authorization: Basic Zm9vOmJhcg==` (`foo:bar`
> > +encoded in base64). And for `tectonic.com` - `Authorization: Bearer
> > +tectonic-token`.
> > +
> > +### rktKind: `dockerAuth`
> > +
> > +This kind of configuration is used to set up necessary credentials
> > +when downloading data from docker indices. The configuration files
> > +should be placed inside either `auth.d` or `docker.d` subdirectories
>
> I was just wondering if we might end up having some more configuration
> kinds related to docker. If you don't think so, then this directory indeed
> doesn't have much sense.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/rkt/pull/767/files#r28622600>.
>